example.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>U&I | Examples</title>
+    <link rel="icon" href="./Components/images/favicon.ico" type="image/x-icon"/>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Recursive:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w==" crossorigin="anonymous" />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+        
+        <nav class="nav-bar shadow-box" id="my-nav-bar">
+            <div class="nav-section">
+                <div id="my-burger-menu" class="burger nav-section-items"> 
+                    <div class="line1"></div>
+                    <div class="line2"></div>
+                    <div class="line3"></div>
+                </div>
+
+                <div class="nav-logo nav-section-items" id="my-logo">
+                    <a class="link-no-style" href="./index.html"><div class="primary-text-color logo-title">U<span class="tertiary-text-color" style="font-size:1rem; margin-right:3px;">&</span>I
+                    </div>
+                    <div class="tertiary-text-color" style="font-size:0.625rem; font-weight:500; letter-spacing: 1px;">LET'S DESIGN TOGETHER</div></a>
+                </div>
+        
+                <ul class="nav-bar-links list-style-none nav-section-items">
+                    <li class="list-inline-item avatar-in-nav-links">
+                        <div class="primary-text-color logo-title">U<span class="tertiary-text-color" style="font-size:1rem; margin-right: 3px;">&</span>I</div>
+                    <div class="tertiary-text-color" style="font-size:0.625rem; font-weight:500; letter-spacing: 1px;">LET'S DESIGN TOGETHER</div>
```

Comment:
Try not to use inline styles, you can make a classes like `font-bold` and `text-small` etc and use them here.

Reply:
I have removed inline styles and created a class for it.